locales/it/index.yml

```diff
@@ -808,6 +808,12 @@ datetimes:
   todayAt: oggi, alle
   notValid: Data non valida.
 payment:
+  alertNoPaymentMethods:
+    title: Aggiungi un metodo per effettuare pagamenti in app
+    message: Il metodo verrà salvato nel Portafoglio, così la prossima volta potrai pagare più velocemente.
+    buttons:
+      ok: Continua
+      ko: Annulla
```

Comment:
A basic translation for the German dictionary could be:

```
alertNoPaymentMethods:
    title: Bitte hinzufügen Sie einen Zahlungsdienstleister, um aus der IO-App zu bezahlen
    message: Der Zahlungsmethode wird in Ihrem Konto Gespeichert, damit Sie nächster Mal schneller bezahlen können.
    buttons:
      ok: Weiter
      ko: Abbrechen
```

My German is pretty rusty, though it should be better than nothing :-)

Reply:
@pietro909 if it's ok for @thisisjp could you add a commit with the proposal locale?

Reply:
https://github.com/pagopa/io-app/pull/3272/commits/0cef0c0b8fa623c23c4b5e5c109c83939c2ac44a